testing/podman/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Simon Ser <simon.ser@intel.com>
+# Maintainer: Simon Ser <simon.ser@intel.com>
+pkgname=podman
+pkgver=1.5.1
+pkgrel=1
+pkgdesc="A daemonless container engine for developing, managing, and running OCI Containers"
+url="https://podman.io/"
+arch="all"
+license="Apache-2.0"
+makedepends="go bash btrfs-progs-dev lvm2-dev gpgme-dev libgpg-error-dev libseccomp-dev ostree-dev"
+depends="btrfs-progs cni-plugins device-mapper-libs go-md2man gpgme iptables ip6tables libassuan libgpg-error libseccomp conmon ostree runc skopeo"
```

Comment:
i think some of the libraries here will be automatically found via so: providers

namely those that are included in -dev packages in $makedepends

Reply:
Also wise to cut this line at 72 columns (before if 72th column land in the middle of a string) for better readability